tests/network-setups/corda/README.md

```diff
@@ -56,69 +59,47 @@ To get the CorDapps from Github Packages you will need to have permission to do
 3) Replace <GITHUB Email> with your email id for github.
 4) Replace <GITHUB Personal Access Token> with your personal access token.
 
-## Running Corda Simple Application with Make
-
-If you are happy with the default configuration, you can use the following make
-targets.
-
--   `make start` builds the required node folder structure for the nodes to run,
-    gets the CorDapps from Github Packages, then starts the Corda nodes.
--   `make start-local` is the same as `make start` except it gets the cordapps
-    from locally built jars from the `corda-simple-app` and `corda-interop-app`
-    repositories (delete `github.properties` if present, else it will try to fetch dependencies from Github Packages).
-    * _Prerequisites_: Before you run this command, build the following in sequence:
-      **Protobufs**
-      ```
-      cd ../../../common/protos-java-kt
-      make build
-      ```
-      **Interoperation CorDapp**
-      ```
-      cd ../../../core/network/corda-interop-app
-      make build-local
-      ```
-      **CLI Client and CorDapp**
-      ```
-      cd ../../../samples/corda/corda-simple-application
-      make build-local
-      ```
--   `make restart-with-new-interop-app` can be used to restart an already running
-    Corda network with new local versions of the `corda-interop-app` CorDapp.
--   `make stop` stops the nodes.
+## Deploying Corda Testnet
+
+- To start the corda networks (`Corda_Network`, and `Corda_Network2`), using CorDapps from Github Packages, refer to the following command:
+  ```bash
+  make start APP_NAME=<simple/house> PROFILE=<1-node/2-nodes/3-nodes>
+  ```
+  Here `simple` refers to `cordaSimpleApplication` and `house` refers to the `fungible-house-token` app.
```

Comment:
Would you rather use `house` or `tokens`? Is the former a well-known term in the Corda world?

Reply:
Both apps manage tokens. So the differentiating term I believe is Simple and House, thus `simple` and `house`.